### ISSUE TITLE
About Image normalization 

### ISSUE THREAD

**Issue by moonchildz** [guanyingc/PS-FCN#10]
I found  that you published a paper named : 
Deep Photometric Stereo for Non-Lambertian Surfaces

and I found that you normalized the images before putting images into network and obtained better result. 

Did you normalize all image which are participated in training step ? Or you just normalized test datasets? 
I tried to normalize the images in dataset(blobby and sculpture) and found that normalization process cannnot output not that great result( like the image below ) 

![image](https://user-images.githubusercontent.com/66664844/88203662-231c5f00-cc85-11ea-95a9-2696b39b0bab.png)

![image](https://user-images.githubusercontent.com/66664844/88203693-2c0d3080-cc85-11ea-8fff-ce9eea5fe525.png)

**Comment by guanyingc** [comment]
Hi, to use data normalization, you have to do normalization during training. 

What do you mean "normalization process cannnot output not that great result"? Are you saying the images after normalization are not good or the results after training are not good?

Also are you using the data normalization code provided in this repository?

**Comment by moonchildz** [comment]
No I am about to train the data soon. 

I made my own normalization code referring the equation you wrote in the paper . 
Do you mean the kaiming-normalization in your code ?

**Comment by guanyingc** [comment]
No, I don't mean kaiming-normalization. I am not sure whether some of the normalized images look like the ones you showed. But it is possible as the appearance of the normalized images depends on the object reflectance.

If you have implemented the normalization strategy introduced in the paper, I think you can just train the model using the normalized images, and after convergence, the model will learn the mapping from image observation to surface normal. But please remember to multiply the scaling factor if the training image number is different from the testing number.

If your code cannot converge, you may try train the model use my code to see if it works.

**Comment by moonchildz** [comment]
Thanks for your advice. I will try it :)
